apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/CustomItemAdapter.java

```diff
@@ -0,0 +1,63 @@
+package com.microsoft.appcenter.sasquatch.activities;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ImageButton;
+import android.widget.ListAdapter;
+import android.widget.TextView;
+
+import com.microsoft.appcenter.sasquatch.R;
+
+import java.util.ArrayList;
+
+public class CustomItemAdapter extends BaseAdapter implements ListAdapter {
+    private ArrayList<String> list = new ArrayList<String>();
```

Comment:
add blank lines before/after fields.

member variables must be named `m{Field}`, like here it has to be `mList` (or s for static).

Reply:
Fix in new commit.